### DISCUSSION TITLE
Welcome to the FIP Discussion Forum!

### DISCUSSION THREAD

**Discussion by kaitlin-beegle** [filecoin-project/FIPs#194] [General]
# What is the FIP Discussion Forum?

The purpose of the FIP Discussion Forum is to provide a space for sharing ideas, thoughts, and processes for enhancing the Filecoin protocol.

Previously, there were two spaces for doing this: Slack, or by opening up an 'Issue' in the FIPs repo.  However, this made it difficult to keep track of and organize conversations. It also led to confusion, as the 'Issues' log became a catch-all place for proposals at very different levels of doneness. 

The FIP Discussion Forum is designed to pilot a use-case that bridges the convenience of Slack with Github's capacity for transparent record-keeping. 

**The goal is to create a space for logging and improving ideas over time.** Community members can brainstorm improvements, support can gather, and resources can be allocated as capacity becomes available. Large, complex proposals (e.g. EVM/WASM development, market funding mechanisms, etc.) can be broken into different discussion threads without getting lost. Community members who are new to the Filecoin ecosystem will also be able to more easily navigate and learn from these forums. 

This forum space will also help to define future community deliberation needs as the Filecoin ecosystem continues to grow and develop. 

# Using the FIP Discussion Forum

First and foremost, the FIP Discussion Forum should be an accesssible space for anyone in the Filecoin ecosystem. This is the space where ideas get credentialed.  No idea is too simple, and no knowledge 'threshhold' is required for participation. 

The only guideline is that this forum be limited to FIP -relevant conversations, i.e., ideas for changing something about how Filecoin works.  General Q&A, onboarding resources, and catch-all conversations should instead be directed to the [Filecoin Community Discussion Forum](https://github.com/filecoin-project/community/discussions).

To begin, community members should think about the scope of their idea and house it in the proper channel: 
* **Announcements** 
    * News about new FIP statuses, governance events, and relevant protocol changes, etc.
* **Enhancements - Governance** 
    * Ideas for improving Filecoin governance, FIP process reviews, etc. 
* **Enhancements - Misc**
    * Got an idea that isn't easily classified? Drop it here! 
* **Enhancements - Organizational** 
    * Ideas for ecosystem operational improvements, terminology changes, programming requests for Filecoin Foundation, etc. 
* **Enhancements - Technical** 
    * Ideas for technical protocol enhancements (the most common type of FIP)
* **General**
    * A space for Web3 and governance conversation topics, etc. 
* **Q&A**
    * Question about Filecoin governance or the FIPs process? Ask it here!

The FIP Discussion Forum is a place for ideas to live until they are ready to become FIP proposals.  Once a FIP proposal is ready to be drafted, authors are asked to do so by opening up an 'Issue' in the FIPs repo. 

![](https://i.imgur.com/gFl2Y6F.png)

Please note that the Forum is an opportunity for sharing and storing ideas until they are ready to become FIP proposals. **This Forum is merely a tool and it does not have to be used.** If you or your team are ready to write a FIP draft and begin the FIP process, you are welcome to post your draft directly in the FIPs Issues log. 

The FIP Issues log is where all active FIP Drafts should live. No idea is considered for acceptance until a FIP Draft is authored and pushed to the Issues log.

**Comment by gdragonHH** [comment]
硬件算力出海，专业技术团队 独立节点，欢迎大家加V了解：Hmcxxxxx
